packages/react/src/components/MultiSelect/FilterableMultiSelect.tsx

```diff
@@ -26,9 +41,179 @@ import { useFeatureFlag } from '../FeatureFlags';
 import { usePrefix } from '../../internal/usePrefix';
 import { FormContext } from '../FluidForm';
 
-const FilterableMultiSelect = React.forwardRef(function FilterableMultiSelect(
+export interface FilterableMultiSelectProps<Item extends ItemBase>
+  extends SortingPropTypes<Item> {
+  /**
+   * Specify a label to be read by screen readers on the container node
+   */
+  'aria-label'?: string;
+
+  className?: string;
+
+  /**
+   * Specify the direction of the multiselect dropdown.
+   */
+  direction?: 'top' | 'bottom';
+
+  /**
+   * Disable the control
+   */
+  disabled?: boolean;
+
+  /**
+   * Additional props passed to Downshift
+   */
+  downshiftProps?: DownshiftProps<Item>;
+
+  /**
+   * Default sorter is assigned if not provided.
+   */
+  filterItems(
+    items: readonly Item[],
+    extra: {
+      inputValue: string | null;
+      itemToString: NonNullable<DownshiftProps<Item>['itemToString']>;
+    }
+  ): Item[];
+
+  /**
+   * Specify whether the title text should be hidden or not
+   */
+  hideLabel?: boolean;
+
+  /**
+   * Provide helper text that is used alongside
+   * the control label for additional help
+   */
+  helperText?: ReactNode;
```

Comment:
Yeah ideally everything expected would be declared in propTypes

Reply:
And would you expect me to add those in too? This one is easy, obviously, but that is not a given in general & I have completely stopped using propTypes a long time ago, so I'm not eager to jump into that if you're fine with `it was wrong until now, it can die with propTypes`